<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Publications | CoEvolution Lab </title> <meta name="author" content="CoEvolution Lab"> <meta name="description" content="Publications are in reversed chronological order."> <meta name="keywords" content="CoEvolutionLab, academic-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/coevolutionlab_logo.png?99809ece260d029add7a156c7678ffcb"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://madhavi-patil.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">CoEvolution</span> Lab </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About </a> </li> <li class="nav-item "> <a class="nav-link" href="/research/">Research </a> </li> <li class="nav-item "> <a class="nav-link" href="/members/">Members </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">Teaching </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">More </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/news/">News</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/contactus/">Contact us</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/joinus/">Join us</a> </div> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">Projects </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description">Publications are in reversed chronological order.</p> </header> <article> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Tao2024-ym" class="col-sm-8"> <div class="title">Atlas of telomeric repeat diversity in Arabidopsis thaliana</div> <div class="author"> Yueqi Tao , Wenfei Xian , Zhigui Bao , Fernando A Rabanal , Andrea Movilli , Christa Lanz , <em>Gautam Shirsekar</em> , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Genome Biology</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Karasov2024-sp" class="col-sm-8"> <div class="title">Continental-scale associations of Arabidopsis thaliana phyllosphere members with host genotype and drought</div> <div class="author"> Talia L Karasov , Manuela Neumann , Laura Leventhal , Efthymia Symeonidi , <em>Gautam Shirsekar</em> , Aubrey Hawks , Grey Monroe , Moisés Exposito-Alonso , Joy Bergelson , and Others </div> <div class="periodical"> <em>Nature Microbiology</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mencia2024-et" class="col-sm-8"> <div class="title">Transposon-triggered epigenetic chromatin dynamics modulate EFR-related pathogen response</div> <div class="author"> Regina Mencia , Agustín L Arce , Candela Houriet , Wenfei Xian , Adrián Contreras , <em>Gautam Shirsekar</em> , <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> , and Pablo A Manavella </div> <div class="periodical"> <em>Nature structural &amp; molecular biology</em>, 2024 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Teasdale2024-it" class="col-sm-8"> <div class="title">Pangenomic context reveals the extent of intraspecific plant NLR evolution</div> <div class="author"> Luisa C Teasdale , Kevin D Murray , Max Collenberg , Adrián Contreras-Garrido , Theresa Schlegel , Leon Ess , Justina Jüttner , Christa Lanz , Oliver Deusch , Joffrey Fitz , Regina Mencia , Rosanne van D Velthoven , Hajk-Georg Drost , <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> , and <em>Gautam Shirsekar</em> </div> <div class="periodical"> <em>bioRxiv</em>, Sep 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Nucleotide-binding leucine-rich repeat (NLR) proteins are a major component of the plant immune system, which directly or indirectly detect molecular signals of pathogen invasion. Despite their critical role, the processes by which NLR genes diversify remain poorly characterised due to the extraordinary sequence, structural, and regulatory variability of NLRs, even among closely related individuals. To understand the evolution of NLR diversity in Arabidopsis thaliana, we leverage graph-based methods to define pangenomic NLR neighbourhoods in 17 genetically diverse genomes. We integrate full-length transcript and transposable element information to exhaustively annotate all intact and degraded NLRs, enabling exploration of the processes that underpin the birth, death and maintenance of NLR diversity within a species. Our main finding is that many uncorrelated mutational processes create NLR diversity, and that there is no single metric that captures on its own the true extent of NLR structural and sequence variation. This immense diversity in plant immune system diversification allows populations to survive the constant onslaught of pathogens, not unlike vertebrate adaptive immunity, where variation is also generated by a variety of complementary mechanisms, albeit at the level of individuals.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Wlodzimierz2023-jo" class="col-sm-8"> <div class="title">Cycles of satellite and transposon evolution in Arabidopsis centromeres</div> <div class="author"> Piotr Wlodzimierz , Fernando A Rabanal , Robin Burns , Matthew Naish , Elias Primetis , Alison Scott , Terezie Mandáková , Nicola Gorringe , Andrew J Tock , Daniel Holland , Katrin Fritschi , Anette Habring , Christa Lanz , Christie Patel , Theresa Schlegel , Maximilian Collenberg , Miriam Mielke , Magnus Nordborg , Fabrice Roux , <em>Gautam Shirsekar</em> , Carlos Alonso-Blanco , Martin A Lysak , Polina Y Novikova , Alexandros Bousios , <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> , and Ian R Henderson </div> <div class="periodical"> <em>Nature</em>, Jun 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Centromeres are critical for cell division, loading CENH3 or CENPA histone variant nucleosomes, directing kinetochore formation and allowing chromosome segregation1,2. Despite their conserved function, centromere size and structure are diverse across species. To understand this centromere paradox3,4, it is necessary to know how centromeric diversity is generated and whether it reflects ancient trans-species variation or, instead, rapid post-speciation divergence. To address these questions, we assembled 346 centromeres from 66 Arabidopsis thaliana and 2 Arabidopsis lyrata accessions, which exhibited a remarkable degree of intra- and inter-species diversity. A. thaliana centromere repeat arrays are embedded in linkage blocks, despite ongoing internal satellite turnover, consistent with roles for unidirectional gene conversion or unequal crossover between sister chromatids in sequence diversification. Additionally, centrophilic ATHILA transposons have recently invaded the satellite arrays. To counter ATHILA invasion, chromosome-specific bursts of satellite homogenization generate higher-order repeats and purge transposons, in line with cycles of repeat evolution. Centromeric sequence changes are even more extreme in comparison between A. thaliana and A. lyrata. Together, our findings identify rapid cycles of transposon invasion and purging through satellite homogenization, which drive centromere evolution and ultimately contribute to speciation.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Vogt*2022-wv" class="col-sm-8"> <div class="title">vcf2gwas: Python API for comprehensive GWAS analysis using GEMMA</div> <div class="author"> Frank Vogt<sup>*</sup> , <em>Gautam Shirsekar<sup>*</sup></em> , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Bioinformatics</em>, Jun 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Vogt2022-ok" class="col-sm-8"> <div class="title">vcf2gwas: Python API for comprehensive GWAS analysis using GEMMA</div> <div class="author"> Frank Vogt , <em>Gautam Shirsekar</em> , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Bioinformatics</em>, Jan 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>MOTIVATION: Genome-wide association study (GWAS) requires a researcher to perform a multitude of different actions during analysis. From editing and formatting genotype and phenotype information to running the analysis software to summarizing and visualizing the results. A typical GWAS workflow poses a significant challenge of utilizing the command-line, manual text-editing and requiring knowledge of one or more programming/scripting languages, especially for newcomers. RESULTS: vcf2gwas is a package that provides a convenient pipeline to perform all of the steps of a traditional GWAS workflow by reducing it to a single command-line input of a Variant Call Format file and a phenotype data file. In addition, all the required software is installed with the package. vcf2gwas also implements several useful features enhancing the reproducibility of GWAS analysis. AVAILABILITY AND IMPLEMENTATION: The source code of vcf2gwas is available under the GNU General Public License. The package can be easily installed using conda. Installation instructions and a manual including tutorials can be accessed on the package website at https://github.com/frankvogt/vcf2gwas. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Shirsekar2021-du" class="col-sm-8"> <div class="title">Multiple sources of introduction of north American arabidopsis thaliana from across Eurasia</div> <div class="author"> <em>Gautam Shirsekar</em> , Jane Devos , Sergio M Latorre , Andreas Blaha , Maique Queiroz Dias , Alba González Hernando , Derek S Lundberg , Hernán A Burbano , Charles B Fenster , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Molecular biology and evolution</em>, Jan 2021 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Lundberg2021-cy" class="col-sm-8"> <div class="title">Host-associated microbe PCR (hamPCR) enables convenient measurement of both microbial load and community composition</div> <div class="author"> Derek S Lundberg , Pratchaya Pramoj Na Ayutthaya , Annett Strauß , <em>Gautam Shirsekar</em> , Wen-Sui Lo , Thomas Lahaye , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Elife</em>, Jan 2021 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Vogt2021-tk" class="col-sm-8"> <div class="title">vcf2gwas-python API for comprehensive GWAS analysis using GEMMA</div> <div class="author"> Frank Vogt , <em>Gautam Shirsekar</em> , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Bioinformatics</em>, Oct 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>MOTIVATION: Genome-wide association study (GWAS) requires a researcher to perform a multitude of different actions during analysis. From editing and formatting genotype and phenotype information to running the analysis software to summarizing and visualizing the results. A typical GWAS workflow poses a significant challenge of utilizing the command-line, manual text-editing and requiring knowledge of one or more programming/scripting languages, especially for newcomers. RESULTS: vcf2gwas is a package that provides a convenient pipeline to perform all of the steps of a traditional GWAS workflow by reducing it to a single command-line input of a Variant Call Format (VCF) file and a phenotype data file. Additionally, all the required software is installed with the package. vcf2gwas also implements several useful features enhancing the reproducibility of GWAS analysis. AVAILABILITY AND IMPLEMENTATION: The source code of vcf2gwas is available under the GNU General Public License. The package can be easily installed using conda. Installation instructions and a manual including tutorials can be accessed on the package website at https://github.com/frankvogt/vcf2gwas. SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Shirsekar2021-do" class="col-sm-8"> <div class="title">Multiple Sources of Introduction of North American Arabidopsis thaliana from across Eurasia</div> <div class="author"> <em>Gautam Shirsekar</em> , Jane Devos , Sergio M Latorre , Andreas Blaha , Maique Queiroz Dias , Alba González Hernando , D Lundberg , H Burbano , C Fenster , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">D Weigel</a> </div> <div class="periodical"> <em>Mol. Biol. Evol.</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Abstract Large-scale movement of organisms across their habitable range, or migration, is an important evolutionary process that can shape genetic diversity and influence the adaptive spread of alleles. Although human migrations have been studied in great detail with modern and ancient genomes, recent anthropogenic influence on reducing the biogeographical constraints on the migration of nonnative species has presented opportunities in several study systems to ask the questions about how repeated introductions shape genetic diversity in the introduced range. We present an extensive overview of population structure of North American Arabidopsis thaliana by studying a set of 500 whole-genome sequenced and over 2,800 RAD-seq genotyped individuals in the context of global diversity represented by Afro-Eurasian genomes. We use methods based on haplotype and rare-allele sharing as well as phylogenetic modeling to identify likely sources of introductions of extant N. American A. thaliana from the native range in Africa and Eurasia. We find evidence of admixture among the introduced lineages having increased haplotype diversity and reduced mutational load. We also detect signals of selection in immune-system-related genes that may impart qualitative disease resistance to pathogens of bacterial and oomycete origin. We conclude that multiple introductions to a nonnative range can rapidly enhance the adaptive potential of a colonizing species by increasing haplotypic diversity through admixture. Our results lay the foundation for further investigations into the functional significance of admixture.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Karasov2020-zo" class="col-sm-8"> <div class="title">What natural variation can teach us about resistance durability</div> <div class="author"> Talia L Karasov , <em>Gautam Shirsekar</em> , Rebecca Schwab , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Current opinion in plant biology</em>, Sep 2020 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Regalado2020-fk" class="col-sm-8"> <div class="title">Combining whole-genome shotgun sequencing and rRNA gene amplicon analyses to improve detection of microbe–microbe interaction networks in plant leaves</div> <div class="author"> Julian Regalado , Derek S Lundberg , Oliver Deusch , Sonja Kersten , Talia Karasov , Karin Poersch , <em>Gautam Shirsekar</em> , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>The ISME journal</em>, Sep 2020 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Karasov2020-ik" class="col-sm-8"> <div class="title">What natural variation can teach us about resistance durability</div> <div class="author"> Talia L Karasov , <em>Gautam Shirsekar</em> , Rebecca Schwab , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Curr. Opin. Plant Biol.</em>, Aug 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Breeding a crop variety to be resistant to a pathogen usually takes years. This is problematic because pathogens, with short generation times and fluid genomes, adapt quickly to overcome resistance. The triumph of the pathogen is not inevitable, however, as there are numerous examples of durable resistance, particularly in wild plants. Which factors then contribute to such resistance stability over millennia? We review current knowledge of wild and agricultural pathosystems, detailing the importance of genetic, species and spatial heterogeneity in the prevention of pathogen outbreaks. We also highlight challenges associated with increasing resistance diversity in crops, both in light of pathogen (co-)evolution and breeding practices. Historically it has been difficult to incorporate heterogeneity into agriculture due to reduced efficiency in harvesting. Recent advances implementing computer vision and automation in agricultural production may improve our ability to harvest mixed genotype and mixed species plantings, thereby increasing resistance durability.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Lundberg2020-ea" class="col-sm-8"> <div class="title">Measuring both microbial load and diversity with a single amplicon sequencing library</div> <div class="author"> Derek S Lundberg , Pratchaya Pramoj Na Ayutthaya , Annett Strauß , <em>Gautam Shirsekar</em> , Wen-Sui Lo , Thomas Lahaye , and <a href="https://en.wikipedia.org/wiki/Detlef_Weigel" rel="external nofollow noopener" target="_blank">Detlef Weigel</a> </div> <div class="periodical"> <em>Cold Spring Harbor Laboratory</em>, May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The microbial population size, or load, in a host is a fundamental metric of colonization by commensals or infection by pathogens. Sequence-based measurement of DNA amount contributed by host and microbes in a sample provides a simple way of measuring microbial load, and it also has the ability to estimate microbiome diversity. Unfortunately, it is also very costly, especially when host DNA greatly outweighs microbial DNA. We introduce a robust two-step PCR strategy to quantify absolute abundance of the host-associated microbiome and describe its composition in a single, cost-effective amplicon library. We demonstrate the accuracy and flexibility of this method across multiple amplicons and hosts, and further present a simple technique that can be used, prior to sequencing, to optimize the host representation in a batch of libraries without a loss of information. ### Competing Interest Statement The authors have declared no competing interest.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Bai2019-ze" class="col-sm-8"> <div class="title">Role of lysine residues of the Magnaporthe oryzae effector AvrPiz-t in effector- and PAMP-triggered immunity</div> <div class="author"> Pengfei Bai , Chan-Ho Park , <em>Gautam Shirsekar</em> , Pattavipha Songkumarn , Maria Bellizzi , and Guo-Liang Wang </div> <div class="periodical"> <em>Mol. Plant Pathol.</em>, Apr 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Magnaporthe oryzae is an important fungal pathogen of both rice and wheat. However, how M. oryzae effectors modulate plant immunity is not fully understood. Previous studies have shown that the M. oryzae effector AvrPiz-t targets the host ubiquitin-proteasome system to manipulate plant defence. In return, two rice ubiquitin E3 ligases, APIP6 and APIP10, ubiquitinate AvrPiz-t for degradation. To determine how lysine residues contribute to the stability and function of AvrPiz-t, we generated double (K1,2R-AvrPiz-t), triple (K1,2,3R-AvrPiz-t) and lysine-free (LF-AvrPiz-t) mutants by mutating lysines into arginines in AvrPiz-t. LF-AvrPiz-t showed the highest protein accumulation when transiently expressed in rice protoplasts. When co-expressed with APIP10 in Nicotiana benthamiana, LF-AvrPiz-t was more stable than AvrPiz-t and was less able to degrade APIP10. The avirulence of LF-AvrPiz-t on Piz-t:HA plants was less than that of AvrPiz-t, which led to resistance reduction and lower accumulation of the Piz-t:HA protein after inoculation with the LF-AvrPiz-t-carrying isolate. Chitin- and flg22-induced production of reactive oxygen species (ROS) was higher in LF-AvrPiz-t than in AvrPiz-t transgenic plants. In addition, LF-AvrPiz-t transgenic plants were less susceptible than AvrPiz-t transgenic plants to a virulent isolate. Furthermore, both AvrPiz-t and LF-AvrPiz-t interacted with OsRac1, but the suppression of OsRac1-mediated ROS generation by LF-AvrPiz-t was significantly lower than that by AvrPiz-t. Together, these results suggest that the lysine residues of AvrPiz-t are required for its avirulence and virulence functions in rice.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Fan2018-hq" class="col-sm-8"> <div class="title">The Monocot-Specific Receptor-like Kinase SDS2 Controls Cell Death and Immunity in Rice</div> <div class="author"> Jiangbo Fan , Pengfei Bai , Yuese Ning , Jiyang Wang , Xuetao Shi , Yehui Xiong , Kai Zhang , Feng He , Chongyang Zhang , Ruyi Wang , Xiangzong Meng , Jinggeng Zhou , Mo Wang , <em>Gautam Shirsekar</em> , Chan Ho Park , Maria Bellizzi , Wende Liu , Jong-Seong Jeon , Ye Xia , Libo Shan , and Guo-Liang Wang </div> <div class="periodical"> <em>Cell Host Microbe</em>, Apr 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Programmed cell death (PCD) plays critical roles in plant immunity but must be regulated to prevent excessive damage. The E3 ubiquitin ligase SPL11 negatively regulates PCD and immunity in plants. We show that SPL11 cell-death suppressor 2 (SDS2), an S-domain receptor-like kinase, positively regulates PCD and immunity in rice by engaging and regulating SPL11 and related kinases controlling defense responses. An sds2 mutant shows reduced immune responses and enhanced susceptibility to the blast fungus Magnaporthe oryzae. Conversely, SDS2 over-expression induces constitutive PCD accompanied by elevated immune responses and enhanced resistance to M. oryzae. SDS2 interacts with and phosphorylates SPL11, which in turn ubiquitinates SDS2, leading to its degradation. In addition, SDS2 interacts with related receptor-like cytoplasmic kinases, OsRLCK118/176, that positively regulate immunity by phosphorylating the NADPH oxidase OsRbohB to stimulate ROS production. Thus, a plasma membrane-resident protein complex consisting of SDS2, SPL11, and OsRLCK118/176 controls PCD and immunity in rice.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Vo2017-yh" class="col-sm-8"> <div class="title">OsWRKY67 Plays a Positive Role in Basal and XA21-Mediated Resistance in Rice</div> <div class="author"> Kieu T X Vo , Chi-Yeol Kim , Trung V Hoang , Sang-Kyu Lee , <em>Gautam Shirsekar</em> , Young-Su Seo , Sang-Won Lee , Guo-Liang Wang , and Jong-Seong Jeon </div> <div class="periodical"> <em>Front. Plant Sci.</em>, Apr 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>WRKY proteins play important roles in transcriptional reprogramming in plants in response to various stresses including pathogen attack. In this study, we functionally characterized a rice WRKY gene, OsWRKY67, whose expression is upregulated against pathogen challenges. Activation of OsWRKY67 by T-DNA tagging significantly improved the resistance against two rice pathogens, Magnaporthe oryzae and Xanthomonas oryzae pv. oryzae (Xoo). Reactive oxygen species (ROS) rapidly accumulated in OsWRKY67 activation mutant lines in response to elicitor treatment, compared with the controls. Overexpression of OsWRKY67 in rice confirmed enhanced disease resistance, but led to a restriction of plant growth in transgenic lines with high levels of OsWRKY67 protein. OsWRKY67 RNAi lines significantly reduced resistance to M. oryzae and Xoo isolates tested, and abolished XA21-mediated resistance, implying the possibility of broad-spectrum resistance from OsWRKY67. Transcriptional activity and subcellular localization assays indicated that OsWRKY67 is present in the nucleus where it functions as a transcriptional activator. Quantitative PCR revealed that the pathogenesis-related genes, PR1a, PR1b, PR4, PR10a, and PR10b, are upregulated in OsWRKY67 overexpression lines. Therefore, these results suggest that OsWRKY67 positively regulates basal and XA21-mediated resistance, and is a promising candidate for genetic improvement of disease resistance in rice.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Park2016-qi" class="col-sm-8"> <div class="title">The E3 Ligase APIP10 Connects the Effector AvrPiz-t to the NLR Receptor Piz-t in Rice</div> <div class="author"> Chan Ho Park , <em>Gautam Shirsekar</em> , Maria Bellizzi , Songbiao Chen , Pattavipha Songkumarn , Xin Xie , Xuetao Shi , Yuese Ning , Bo Zhou , Pavinee Suttiviriya , Mo Wang , Kenji Umemura , and Guo-Liang Wang </div> <div class="periodical"> <em>PLoS Pathog.</em>, Mar 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Although nucleotide-binding domain, leucine-rich repeat (NLR) proteins are the major immune receptors in plants, the mechanism that controls their activation and immune signaling remains elusive. Here, we report that the avirulence effector AvrPiz-t from Magnaporthe oryzae targets the rice E3 ligase APIP10 for degradation, but that APIP10, in return, ubiquitinates AvrPiz-t and thereby causes its degradation. Silencing of APIP10 in the non-Piz-t background compromises the basal defense against M. oryzae. Conversely, silencing of APIP10 in the Piz-t background causes cell death, significant accumulation of Piz-t, and enhanced resistance to M. oryzae, suggesting that APIP10 is a negative regulator of Piz-t. We show that APIP10 promotes degradation of Piz-t via the 26S proteasome system. Furthermore, we demonstrate that AvrPiz-t stabilizes Piz-t during M. oryzae infection. Together, our results show that APIP10 is a novel E3 ligase that functionally connects the fungal effector AvrPiz-t to its NLR receptor Piz-t in rice.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Chakraborty2016-cx" class="col-sm-8"> <div class="title">Quantification of hydrogen peroxide in plant tissues using Amplex Red</div> <div class="author"> Sourav Chakraborty , Amy L Hill , <em>Gautam Shirsekar</em> , Ahmed J Afzal , Guo-Liang Wang , David Mackey , and Pierluigi Bonello </div> <div class="periodical"> <em>Methods</em>, Oct 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Reactive oxygen species (ROS) are by-products of photosynthesis and respiration in plant tissues. Abiotic and biotic stressors also induce the production and temporary accumulation of ROS in plants, including hydrogen peroxide (H2O2), whereby they can act as secondary messengers/chemical mediators in plant defense signaling and lead to programmed cell death. H2O2 acts as a hub for critical information flow in plants. Despite such key roles in fundamental cellular processes, reliable determination of H2O2 levels in plant tissues is hard to achieve. We optimized an Amplex Red-based quantitation method for H2O2 estimation from plant tissue lysate. The standard limit of detection and quantitation was determined as 6 and 18picomol respectively. In this study we also quantified constitutive and/or induced levels of H2O2 in three model plants, Pinus nigra (Austrian pine), Oryza sativa (rice), and Arabidopsis thaliana. Overall, assay sensitivity was in the nmolg-1 FW range. Commonly used additives for H2O2 extraction such as activated charcoal, ammonium sulfate, perchloric acid, polyvinylpolypyrrolidone, and trichloroacetic acid either degraded H2O2 directly or interfered with the Amplex Red assay. Finally, We measured stability of Amplex Red working solution over one month of storage at -80°C and found it to be significantly stable over time. With appropriate modifications, this optimized method should be applicable to any plant tissue.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Weis2016-uo" class="col-sm-8"> <div class="title">Temporal patterns of damage and decay kinetics of DNA retrieved from plant herbarium specimens</div> <div class="author"> Clemens L Weiß , Verena J Schuenemann , Jane Devos , <em>Gautam Shirsekar</em> , Ella Reiter , Billie A Gould , John R Stinchcombe , Johannes Krause , and Hernán A Burbano </div> <div class="periodical"> <em>R. Soc. Open Sci.</em>, Jun 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Herbaria archive a record of changes of worldwide plant biodiversity harbouring millions of specimens that contain DNA suitable for genome sequencing. To profit from this resource, it is fundamental to understand in detail the process of DNA degradation in herbarium specimens. We investigated patterns of DNA fragmentation and nucleotide misincorporation by analysing 86 herbarium samples spanning the last 300 years using Illumina shotgun sequencing. We found an exponential decay relationship between DNA fragmentation and time, and estimated a per nucleotide fragmentation rate of 1.66 × 10(-4) per year, which is six times faster than the rate estimated for ancient bones. Additionally, we found that strand breaks occur specially before purines, and that depurination-driven DNA breakage occurs constantly through time and can to a great extent explain decreasing fragment length over time. Similar to what has been found analysing ancient DNA from bones, we found a strong correlation between the deamination-driven accumulation of cytosine to thymine substitutions and time, which reinforces the importance of substitution patterns to authenticate the ancient/historical nature of DNA fragments. Accurate estimations of DNA degradation through time will allow informed decisions about laboratory and computational procedures to take advantage of the vast collection of worldwide herbarium specimens.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mishra2015-cq" class="col-sm-8"> <div class="title">Sources of resistance to Indian pathotypes of Puccinia graminis tritici and P. triticina in durum wheat</div> <div class="author"> A N Mishra , <em>G S Shirsekar</em> , S Yadav , and K Kaushal </div> <div class="periodical"> <em>Plant</em>, Jun 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Stem rust and leaf rust are important diseases affecting durum wheat production in India. Inheritance and extent of diversity in resistance were studied in five durum genotypes, viz. ʽB 662’, ʽ ED 2398‐A’, ʽ HG 110’, ʽ IWP 5019’and ʽLine 1172ʼ using Pgt pathotypes 40A …</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Shirsekar2014-zg" class="col-sm-8"> <div class="title">Identification and characterization of suppressor mutants of spl11- mediated cell death in rice</div> <div class="author"> <em>Gautam S Shirsekar</em> , Miguel E Vega-Sanchez , Alicia Bordeos , Marietta Baraoidan , Andrew Swisshelm , Jiangbo Fan , Chan Ho Park , Hei Leung , and Guo-Liang Wang </div> <div class="periodical"> <em>Mol. Plant. Microbe. Interact.</em>, Jun 2014 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Lesion mimic mutants have been used to dissect programmed cell death (PCD) and defense-related pathways in plants. The rice lesion-mimic mutant spl11 exhibits race nonspecific resistance to the bacterial pathogen Xanthomonas oryzae pv. oryzae and the fungal pathogen Magnaporthe oryzae. Spl11 encodes an E3 ubiquitin ligase and is a negative regulator of PCD in rice. To study the regulation of Spl11-mediated PCD, we performed a genetic screen and identified three spl11 cell-death suppressor (sds) mutants. These suppressors were characterized for their resistance to X. oryzae pv. oryzae and M. oryzae and for their expression of defense-related genes. The suppression of the cell-death phenotypes was generally correlated with reduced expression of defense-related genes. When rice was challenged with avirulent isolates of M. oryzae, the disease phenotype was unaffected in the sds mutants, indicating that the suppression might be Spl11-mediated pathway specific and may only be involved in basal defense. In addition, we mapped one of the suppressor mutations to a 140-kb interval on the long arm of rice chromosome 1. Identification and characterization of these sds mutants should facilitate our efforts to elucidate the Spl11-mediated PCD pathway.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2012</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Park2012-cv" class="col-sm-8"> <div class="title">The Magnaporthe oryzae Effector AvrPiz-t Targets the RING E3 Ubiquitin Ligase APIP6 to Suppress Pathogen-Associated Molecular Pattern–Triggered Immunity in …</div> <div class="author"> C H Park , S Chen , <em>G Shirsekar</em> , and B Zhou </div> <div class="periodical"> <em>The Plant</em>, Jun 2012 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Although the functions of a few effector proteins produced by bacterial and oomycete plant pathogens have been elucidated in recent years, information for the vast majority of pathogen …</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Liu2012-lh" class="col-sm-8"> <div class="title">The U-Box E3 ligase SPL11/PUB13 is a convergence point of defense and flowering signaling in plants</div> <div class="author"> Jinling Liu , Wei Li , Yuese Ning , <em>Gautam Shirsekar</em> , Yuhui Cai , Xuli Wang , Liangying Dai , Zhilong Wang , Wende Liu , and Guo-Liang Wang </div> <div class="periodical"> <em>Plant Physiol.</em>, Sep 2012 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Park2012-ne" class="col-sm-8"> <div class="title">The Magnaporthe oryzae effector AvrPiz-t targets the RING E3 Ubiquitin Ligase APIP6 to suppress pathogen-associated molecular pattern–triggered immunity in rice</div> <div class="author"> C H Park , S Chen , <em>G Shirsekar</em> , B Zhou , and C H Khang </div> <div class="periodical"> <em>The Plant</em>, Sep 2012 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Although the functions of a few effector proteins produced by bacterial and oomycete plant pathogens have been elucidated in recent years, information for the vast majority of pathogen effectors is still lacking, particularly for those of plant-pathogenic fungi. Here, we …</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2011</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mishra2011-ps" class="col-sm-8"> <div class="title">Diversity for resistance to stem and leaf rusts in Indian wheat germplasm</div> <div class="author"> A N Mishra , S R Yadav , and <em>G S Shirsekar</em> </div> <div class="periodical"> <em>Indian J. Clin. Biochem.</em>, Sep 2011 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>In India, three wheat species-Triticum aestivum L. emend. Fiori &amp; Paol (bread wheat), T. durum Desf.(durum wheat) and T. dicoccum Schrank (khapli wheat) are under cultivation, which are attacked by stem rust (Puccinia graminis Pers. f. sp. tritici Eriks. &amp; Henn.) and leaf …</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Shirsekar2010-uy" class="col-sm-8"> <div class="title">Role of ubiquitination in plant innate immunity and pathogen virulence</div> <div class="author"> <em>G Shirsekar</em> , L Dai , Y Hu , X Wang , and L Zeng </div> <div class="periodical"> <em>J. Plant Biol.</em>, Sep 2010 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Plant diseases are a major constraint for stable crop production in the world. Plants are constantly threatened by different pathogens and have developed an array of mechanisms to defend themselves. A growing body of evidence indicates that ubiquitination, which is one …</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2009</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mishra2009-wg" class="col-sm-8"> <div class="title">Protocols for evaluating resistance to leaf and stem rusts in durum and bread wheats</div> <div class="author"> A N Mishra , <em>G S Shirsekar</em> , S R Yadav , and V G Dubey </div> <div class="periodical"> <em>Indian</em>, Sep 2009 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>see more details-12-5 (29R45), 162-2 (93R39), 12-2 (1R5), 104-2 (21R55), 162-3 (29R7), 11 (0R8) and 106 (0R9), and stem rust pathotypes-117-6 (37G19), 117A (38G2), 117-1 (166G2), 117A-1 (38G18) and 117-3 (167G3) were found appropriate for evaluating …</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2005</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mishra2005-bu" class="col-sm-8"> <div class="title">Genetic basis of seedling‐resistance to leaf rust in bread wheat ’Thatcher’</div> <div class="author"> A N Mishra , K Kaushal , <em>G S Shirsekar</em> , and S R Yadav </div> <div class="periodical"> <em>Plant</em>, Sep 2005 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The bread wheat cultivar ’Thatcher’is documented to carry the gene Lr22b for adult‐plant resistance to leaf rust. Seedling‐resistance to leaf rust caused by Puccinia triticina in the bread wheat cultivar ’Thatcher’, the background parent of the near‐isogenic lines for leaf rust …</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mishra2005-zn" class="col-sm-8"> <div class="title">The linkage between the stem rust resistance gene Sr2 and pseudo‐black chaff in wheat can be broken</div> <div class="author"> A N Mishra , K Kaushal , S R Yadav , and <em>G S Shirsekar</em> </div> <div class="periodical"> <em>Plant</em>, Sep 2005 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Recessively inherited gene Sr2 has provided the basis of durable resistance to stem rust (caused by Puccinia graminis tritici) in wheat (Triticum aestivum L.) worldwide. The associated earhead and stem melanism or ’pseudo‐black chaff’is generally used as a …</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Mishra2005-it" class="col-sm-8"> <div class="title">A leaf rust resistance gene, different from Lr34, associated with leaf tip necrosis in wheat</div> <div class="author"> A N Mishra , K Kaushal , S R Yadav , and <em>G S Shirsekar</em> </div> <div class="periodical"> <em>Plant</em>, Sep 2005 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The gene Lr34 has contributed to durable resistance to leaf rust caused by Puccinia triticina in wheat worldwide. The closely associated leaf tip necrosis is generally used as the gene’s marker. Lr34 has been postulated in many Indian bread wheat cultivars including ’C 306’ …</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 CoEvolution Lab. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: May 16, 2025. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>